db/schema.rb

```diff
@@ -73,14 +60,15 @@
   end
 
   create_table "homework_submissions", force: :cascade do |t|
+    t.bigint "user_id", null: false
```

Comment:
wait shouldn't we be using `course_session_participant` in lieu of `user_id` here? am I mixed up or does the migration disagree?